Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```diff
@@ -394,7 +394,7 @@
 # Tastes like crust, tomato, cheese, sausage, sass.
 
 - type: entity
-  name: Hawaiian pizza
+  name: hawaiian pizza
```

Comment:
Hawaii is a place, and therefore a proper noun. Proper nouns should remain capitalized.